src/main/java/com/solvd/webtesting/elements/ProductMenuBar.java

```diff
@@ -0,0 +1,33 @@
+package com.solvd.webtesting.elements;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractUIObject;
+import com.solvd.webtesting.page.ProductPage;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class ProductMenuBar extends AbstractUIObject {
+
+    @FindBy(css = ".styles_promoItem__3H-t7")
+    private List<ExtendedWebElement> itemProducts;
+
+    public ProductMenuBar(WebDriver driver, SearchContext searchContext) {
+        super(driver, searchContext);
+    }
+
+    public ProductPage selectProduct(String productName) {
+        ExtendedWebElement requiredProduct = itemProducts.stream()
+                .filter(product -> product.getText().equals(productName))
+                .findFirst()
+                .orElseThrow();
+        requiredProduct.click();
+        return new ProductPage(getDriver());
+    }
+
+    public List<ExtendedWebElement> getItemProducts() {
+        return itemProducts;
+    }
+}
```

Comment:
Please don't return ExtendedWebElement, if use need to do some manipulations with elements, u need to do this on page where it present.